controller/controller.go

```diff
@@ -640,7 +642,7 @@ waitForLock:
 			if err != nil {
 				logger.WithError(err).Error("error when updating scheduler and waiting for new pods to run")
 				for _, podToDelete := range newPods {
-					deletePod(l, mr, clientset, redisClient.Client, configYAML.Name, podToDelete.GetName())
+					deletePod(l, mr, clientset, redisClient.Client, configYAML.Name, configYAML.Game, podToDelete.GetName(), "update")
```

Comment:
Same here. The pods are being deleted not because of the update, but because some error occurred during update.